mp/src/game/client/momentum/ui/leaderboards/ClientTimesDisplay.cpp

```diff
@@ -182,6 +185,7 @@ void CClientTimesDisplay::ShowPanel(bool bShow)
 
     if (bShow)
     {
+        engine->ClientCmd_Unrestricted("gameui_preventescapetoshow\n");
```

Comment:
I think this should be called when the leaderboards is toggled open:

https://github.com/momentum-mod/game/blob/d854942da710c8f22bdf3cb34815d7d32b9e4457/mp/src/game/client/momentum/ui/leaderboards/ClientTimesDisplay.cpp#L210

Since when it's not toggled open, you just let go of your leaderboards key (TAB) to close it.